### ISSUE TITLE
German translation: How to treat/translate composita containing "Django Girls"

### ISSUE THREAD

**Issue by das-g** [DjangoGirls/tutorial#1421]
### Issue description

In English, one can just chain words with spaces in-between to build new expressions. E.g., "programming workshop".

In German, you can do the same by conjoining the words without spaces to a single word or by hyphenating them. E.g., "Programmierkurs" or "Programmier-Kurs".

So far, so good. Though in the German translation of the Django Girls tutorial (and probably also in other German-language Django Girls materials) we treat "Django Girls" as kind of a brand name that we use unchanged. This brings the above conventions into conflict with each other when trying to translate compound expressions (composita) that have "Django Girls" as one of their parts, like

* Django Girls tutorial
* Django Girls event
* Django Girls workshop

How should we translate these?

### Language

German (de)

### Operating system

N/A

**Comment by das-g** [comment]
I think the options we can choose from are:

1. > Django Girls Veranstaltungen
2. > Django-Girls-Veranstaltungen
3. > "Django Girls"-Veranstaltungen
4. > DjangoGirls Veranstaltungen
5. > DjangoGirls-Veranstaltungen

I dislike 1 and 4, because they use the English way of concatenating words with spaces, which is frowned upon when done in German or even deemed incorrect there.

The name/brand used on djangogirls.org is "Django Girls", _with_ a space, not without one like in the GitHub group name github.com/DjangoGirls. While that name/brand, including the space, is derived from English, it's AFAIK used in all languages alike, and definitely in German. That's a reason to avoid 4 and 5.

So for translating compound expressions (composita) like "Django Girls event", that leaves us with options 2 (the all-hyphenated variant) and 3 (the quotation-marks-and-hyphen variant).

**Comment by das-g** [comment]
Do you agree with the 5 options? Do you agree with my analysis of them? If so, do you prefer 2 or 3?

**Comment by normade** [comment]
I agree with the options and also with your explanation. For me personally, using quotation marks feels to some extent like the quotated thing is something which doesn't really exist, is ironic or something artificial, because this is what I use quotation marks for most of the time. So taken all your arguments, for me the second option is the best.

**Comment by lisaq** [comment]
Adding a hyphen or quotation marks changes the branding; you couldn't do that in English, and the name "Django Girls" is in English. So to me, only 1 makes sense, even in German (but of course I'm not a native German speaker so take that with a grain of salt I guess).

**Comment by normade** [comment]
So 3) turns out to be the best compromise, right? Actually the way, you suggested anyway @das-g.
And if Django Girls needs to be translated without a following noun, e.g. being subject of a sentence, would we use the quotation marks in that case too?

**Comment by das-g** [comment]
> And if Django Girls needs to be translated without a following noun, e.g. being subject of a sentence, would we use the quotation marks in that case too?

No, when not part of a compound term, it'd be just a foreign-language brand name, used as-is (i.e. with the space) and wouldn't require quotation marks. The quotation marks in the compound case are required because
> Django Girls-Veranstaltung

would bind the word "Girls" stronger to "Veranstaltung" than to "Django", leading to a wrong parsing of the meaning. (A "girls event" that is about "Django". Alright, that's not completely wrong, but there may be cases where a bigger semantic difference results, and even in this case it doesn't give the intended focus.)

> So 3) turns out to be the best compromise, right?

I guess so, though I admit it isn't exactly pretty.

I had hoped to get a wider range of opinions from the translators to German (and maybe alternatives I hadn't thought of), but I guess you (@normade) and me are the only currently active ones?

Regarding whether adding a hyphen (only for use in compound terms in German) distorts the brand too much, maybe someone from the core team / from the Django Girls Foundation can chip in?

**Comment by normade** [comment]
@das-g yes, I have also the impression, that at the moment we are representing the German language translation team :-D
Anyways, I have no strong feelings here, but seeing the arguments given, 3) seems to be the best solution so far and if the brand is not part of a compound term, we should just leave it as it is.

**Comment by lisaq** [comment]
Hrm..so we can use either incorrect German or incorrect branding. Yes I think someone from the Foundation should weigh in but I think none of them are German-speaking? This is really hard to understand as a non-native speaker, especially since in English there is a tendency to respect the spelling of loan words (ex. jalapeños) and proper nouns always trump spelling/grammar anyway.

>There's even a (usually tongue-in-cheek) derogatory term for it and websites making fun of that, e.g. https://deppenleerzeichen.de/
Stuff like this is why people give up on learning German 😢

**Comment by das-g** [comment]
> Stuff like this is why people give up on learning German :cry:

Don't give it up! :hugs:
<details><summary>Explanation (click to expand)</summary>

> Separating German compound words with spaces usually just leads to "wrong" grammar, but sometimes instead to a grammatically correct sentence with different (often absurd and funny*) meaning. The mocking by the mentioned website is intended to point that out and to make fun of that way of writing, but explicitly **not** of the writers. Another goal of the people using the term may be to educate others (probably mostly other native speakers) about the correct usage, but I sure hope it's far from most to berate foreign speakers for understandably making mistakes.
> 
> Also note that many native speakers of German, while they may value correct grammar, frown upon such mocking even if it's allegedly not targeted at persons. Others see such mocking as a playful way to appreciate language and its versatility and subtleties (which [languages other than German have, too](https://www.xkcd.com/37/)!), where minor and innocent-seeming errors can change the meaning completely. Of course, like in any society, there are also some few who actually use the term as a serious derogatory term. Ignore them. Hater's gonna hate.
> 
> In either case, none of these should deter anyone from learning a foreign language.
>
> I hesitated to mention the term and the website in my comment above because I consider it somewhat rude, but I came to the conclusion that their existance nicely illustrates that spaces in compound terms is indeed actually considered wrong grammar in German.
> 
> ---
> \* Take for example https://deppenleerzeichen.de/2018/01/14/wenn-saenger-schiefe-platten-aufnehmen/
> 
> The intended meaning was
> > six-part set of 'Sänger' [a brand**] slate [the material] boards
> 
> But it reads like
> > Singer [someone who sings] of off-key discs [read: records] "set" [incomprehensible - may be a fantasy verb] in six parts.
> 
> \** probably derived from a family name, which in turn may be derived from the singer profession

</details>

(Displayed collapsed because that reply is kinda off-topic for the tutorial and this GitHub issue. But I didn't want to just let your understandable sentiment sit here, uncommented.)

**Comment by normade** [comment]
Any news here, @das-g?

**Comment by das-g** [comment]
Oh yeah, sorry for not reporting back. I got a response to my mail shortly after from @GirlGeekUpNorth.

On 2018-11-18, Claire Wicher wrote:
> Hi Raphael,
>
> As Django Girls is the brand name, this is not usually translated into other languages :)
>
> Rainbows, Sparkles, Kittens and Unicorns
>
> :rainbow::sparkles::cat::unicorn:
>
> Claire Wicher
> Django Girls Awesomeness Ambassador :star2:

Not quite sure what to make of that, though. "Not usually" doesn't sound like it's unthinkable to do adaptions to accommodate the grammar of the target language. Also, we're not considering to "translate" the "Django Girls" brand to German.<sup>1</sup> (That'd be "Django-Mädchen" and would in my opinion seem rather silly.) Rather, we're contemplating how to treat the brand in the context of compound terms, which in German often have to be single words without spaces.

So, uh, @GirlGeekUpNorth, do you (or the foundation) want to chime in with a more precise guideline, or should we (tutorial maintainers and translators) just decide among ourselves?

---
<sup>1</sup>In Esperanto however, it is much more usual to translate non-Esperanto brand names (or [used to be, at least](https://esperanto.stackexchange.com/a/5128/1576)), and it's what a fellow translator [is suggesting](https://crowdin.com/translate/django-girls-tutorial/289/en-eo#8470) for the (just started) Esperanto translation of the tutorial. But that just as an aside. If needed, we'll file a separate issue for _that_ decision, as the language-cultural tradition is different between Esperanto and German.
